scripts/rest_examples/reservation-create-azure.http

```diff
@@ -0,0 +1,14 @@
+// @no-log
+POST http://{{hostname}}:{{port}}/{{prefix}}/reservations/azure HTTP/1.1
+Content-Type: application/json
+X-Rh-Identity: {{identity}}
+
+{
+  "name": "azure-linux-us-east",
+  "source_id": "5",
+  "image_id": "/subscriptions/4b9d213f-712f-4d17-a483-8a10bbe9df3a/resourceGroups/oezr/providers/Microsoft.Compute/images/composer-api-00e391e7-e6c8-4283-b94a-ea719a58bc05",
+  "amount": 1,
+  "instance_size": "Standard_B1ls",
+  "pubkey_id": 2,
+  "poweroff": true
+}
```

Comment:
Can you please parametrize source_id (already there) and pubkey_id (not yet present [in the parameters](https://github.com/RHEnVision/provisioning-backend/blob/main/scripts/rest_examples/http-client.env.json) I think it is worth adding). Maybe image_id as well?